packages/components/src/font-size-picker/stories/index.js

```diff
@@ -71,45 +78,37 @@ export const withSlider = () => {
 			slug: 'big',
 			size: 26,
 		},
-	] );
-	const fallbackFontSize = number( 'Fallback Font Size - Slider Only', 16 );
-	return (
-		<FontSizePickerWithState
-			fontSizes={ fontSizes }
-			fallbackFontSize={ fallbackFontSize }
-			withSlider
-		/>
-	);
+	],
+	initialValue: 16,
+	withSlider: false,
 };
 
-export const withoutCustomSizes = () => {
-	const fontSizes = object( 'Font Sizes', [
-		{
-			name: 'Small',
-			slug: 'small',
-			size: 12,
-		},
-		{
-			name: 'Normal',
-			slug: 'normal',
-			size: 16,
-		},
-		{
-			name: 'Big',
-			slug: 'big',
-			size: 26,
+export const WithSlider = FontSizePickerWithState.bind( {} );
+WithSlider.args = {
+	...Default.args,
+	fallbackFontSize: 16,
+	initialValue: undefined,
+	withSlider: true,
+};
+
+export const WithCustomSizesDisabled = FontSizePickerWithState.bind( {} );
+WithCustomSizesDisabled.args = {
+	...Default.args,
+	disableCustomFontSizes: true,
+};
+WithCustomSizesDisabled.parameters = {
+	docs: {
+		description: {
+			story:
+				'When disabled, the user will only be able to pick one of the predefined sizes passed in `fontSizes`.',
 		},
-	] );
-	return (
-		<FontSizePickerWithState
-			fontSizes={ fontSizes }
-			disableCustomFontSizes
-		/>
-	);
+	},
```

Comment:
Do you think there's any chance of breaking our story description in case Storybook updates their APIs? The webpack loader that you linked to hasn't been updated in more than a year

Anyway, we can definitely experiment with it in a separate PR! (although definitely not high priority as it's just a minor improvement)

Reply:
The loader code is pretty trivial and there's a [reasonable chance](https://github.com/storybookjs/storybook/issues/8527) that it will become a built-in feature, so I'm not _too_ concerned about that. I might even try keeping the loader code local, rather than as an external dep.